### ISSUE TITLE
[component name | framework | build | icons | theming | localization | i18n | typescript]: Input

### ISSUE THREAD

**Issue by Garima123639** [SAP/ui5-webcomponents#10249]
### Bug Description

This is in reference to this bug which has been closed and I want to reopen it
https://github.com/SAP/ui5-webcomponents/issues/10231

### Affected Component

Input

### Expected Behaviour

there should be a handler similar to onSuggestionItemSelect which existed in previous versions to handle the selection of item from suggestions.

### Steps to Reproduce

Use Input component with onSelect and having showSuggestions set to true.
Select something from the suggestions either through keyboard down key and press enter or directly mouse click
the handler assigned to onSelect is not fired appropriately

### Priority

None

### UI5 Web Components Version

Version 2

### Browser

Chrome, Edge, Firefox, Safari

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @Garima123639,

Thank you for your feedback. I understand you are looking for a way to handle specifically when the user selects the element from the suggested items in the dropdown through either the keyboard enter or click. 

With ver.2 of ui5 WebComponents, we introduced a breaking change: the suggestion-item-select event is no longer present for ui5-input component.
If you have previously listened to 'suggestion-item-select' event to detect which suggestion item has been selected by the user: 
```js
input.addEventListener("ui5-suggestion-item-select", event => { 
	const suggestionItem = event.detail.item;
	// do something with the suggestion item
});
```
Now you can detect which suggestion item was selected if you attach to 'selection-change event', keep the selected item in a variable and during 'change' event check if the input value matches the last selected item: 
```js
let suggestionItem;

input.addEventListener("ui5-selection-change", (event) => {
   suggestionItem = event.detail.item; 		 
});

input.addEventListener("ui5-change", (event) => {
  if(event.target.value && suggestionItem && 
     (event.target.value === suggestionItem.text)){
    // do something with the suggestion item
  }
});
```

Basically, what was previously achievable with suggestion-item-select should now be done by combining the selection-change and change events. 
Let me know if this helps or if you have any other questions.

Best regards,
Siyana
